Makefile

```diff
@@ -0,0 +1,40 @@
+PROJECT     := airflow_declarative
+PYTHON_VENV := python3
+PYTHON      := python
+
+.PHONY: venv
+venv:
+	$(PYTHON_VENV) -m venv venv
+
+.PHONY: develop
+develop:
+	pip install --upgrade setuptools pip wheel setuptools-pkg
+	# Airflow since 1.10 requires an env var to be set before installation:
+	#
+	# RuntimeError: By default one of Airflow's dependencies installs
+	# a GPL dependency (unidecode). To avoid this dependency set
+	# SLUGIFY_USES_TEXT_UNIDECODE=yes in your environment when you install
+	# or upgrade Airflow. To force installing the GPL version
+	# set AIRFLOW_GPL_UNIDECODE
+	SLUGIFY_USES_TEXT_UNIDECODE=yes pip install 'apache-airflow<1.10'
```

Comment:
That's looks strange since project declares support of any apache-airflow package while here we force to not install latest releases. Can we have a single source of truth about our dependencies?

Reply:
My initial idea was to have the latest version of package installed as a requirement for this project.

I added this line when I discovered that on the latest version of airflow the tests fail, so apparently leaving it like that was not an option. Limiting the version in setup.py doesn't look appealing as well, because a plugin should not decide which version of airflow to install.

Another issue is that when airflow is installed along with this plugin there're some version clashes between the airflow's deps and our deps (i.e. jinja2). When airflow is installed separately, this issue doesn't occur.

I suppose we can remove the explicit dependency from airflow in setup.py, but the plugin does import some stuff from airflow, so technically it does require airflow to be installed.

I would suggest to keep it as it is now.

Reply:
Ok, I have no better ideas for now. Let's keep it until something better comes on mind.